### PR TITLE
R&D Update to TEMPLATE

### PR DESCRIPTION
**Pull request by vbabineau**
Let me know if the things I have added are too similar to the report.
Also idk if the parenthesis part is clear.

### CODE REVIEW

**Comment by vbabineau** [comment]
Also maybe the naming convention isnt the best I realize

File: docs/REPORT.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Template for R&D Reports
+These are the guidelines to follow when proposing new designs or studies.
+> The main focus is on the take-away and conclusion of a report/study.
+### Max of three active designs at any time.
+### File naming convention
+
+Use the format: _report_YYMMDD_part_studytype#.md_
```
Comment (FaizChishtie) [inline docs/REPORT.md:7]: remove the _ after the md or ppl might name it that hahah
> _report_YYMMDD_part_studytype#.md_

File: docs/TEMPLATE.md [hunk 2]
```diff
@@ -33,6 +40,20 @@ Use the format: _build_log_YYMMDD.md_
...
     
+    ## Tests Ran
+    
+    *%Name of Test%
+     * %Detail 1%
+     * %Detail 2%
+    
+    ## Discussion 
+    
+    * %Was the design succesfull?%
```
Comment (FaizChishtie) [inline docs/TEMPLATE.md:51]: small typo: successful*

**Review by FaizChishtie — APPROVED** [review]
looks good - fixes are kinda tiny - merge whenever you want!!

**Pull request merged**